test/namenode/MkdirTest.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 Rice University, COMP 413 2017
+
+#include <limits>
+#include "NameNodeTest.h"
+
+
+TEST_F(NamenodeTest, mkdirDepth1) {
+    std::string src = "/testing/test_mkdir";
+    hadoop::hdfs::MkdirsRequestProto mkdir_req;
+    hadoop::hdfs::MkdirsResponseProto mkdir_resp;
+    mkdir_req.set_createparent(true);
+    mkdir_req.set_src(src);
+    ASSERT_EQ(client->mkdir(mkdir_req, mkdir_resp),
+              zkclient::ZkNnClient::MkdirResponse::Ok);
+    ASSERT_TRUE(mkdir_resp.result());
+    ASSERT_TRUE(client->file_exists(src));
+    LOG(DEBUG) << "Finished all asserts";
+}
+
+TEST_F(NamenodeTest, mkdirDepth1024) {
```

Comment:
this test is failing on CI, maybe reduce the log output?

Reply:
Automatically setting a default log verbosity level on CI of error and warnings only is something I've thought about implementing. Travis will hard-fail a build if its log output is more than 4 MB and it seems like we are testing that limit.

Reply:
I think that's a good idea.